Cargo.toml

```diff
@@ -148,6 +148,7 @@ crt0stack = { version = "0.1.0", default-features = false }
 dirs = { version = "4.0.0", default-features = false }
 drawbridge-client = { version = "0.2.2", default-features = false }
 drawbridge-server = { version = "0.2.2", default-features = false }
+drawbridge-type = { version = "0.2.2", default-features = false } # Temporarily import Drawbridge types for path-related functionality
```

Comment:
There's already a comment about it here https://github.com/enarx/enarx/pull/2245/files/0d9af97bfc4167b1183b0e6ecb9c5b397ca4dc9e..5aa4ef4e72472cef37b8c345295b626a7c0b39cb#diff-48cf782b40cee5697b7f9aebbfc3e82cf9295b092674fba46596aa9f47242278R16

Reply:
You probably have not expanded the diff in `enarx-config`. This is the issue https://github.com/enarx/enarx/issues/2367
Once we have that, this will be removed, since filenames will not be a concern anymore

Reply:
Actually, this made sense in earlier implementation, but probably does not any more. I will rework the types here